src/main/scala/com/bradbrok/GPIOInterface.scala

```diff
@@ -43,12 +25,12 @@ class GPIOInterface {
   val pin15 = gpio.provisionDigitalOutputPin(RaspiPin.GPIO_15, "pin15")
   val pin16 = gpio.provisionDigitalOutputPin(RaspiPin.GPIO_16, "pin16")
 
-  def setHigh(pins: List): Unit ={
+  def setHigh(pins: List[GpioPin]): Unit = {
```

Comment:
Added type of the content of the list here and in the `setLow` method to get the compiler to stop complaining.